src/lib/cli.ts

```diff
@@ -61,6 +69,23 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+type Update = { // eslint-disable-line no-unused-vars
```

Comment:
Then we should use that package instead of doing this. @qzhou1607 can you please add it as a dev dependency?